L1Trigger/GlobalCaloTrigger/test/testElectronSorter.cpp

```diff
@@ -320,6 +321,7 @@ void convertToGct(EmInputCandVec candidates) {
         } else {
           eta = 3;
         }
+        [[fallthrough]];
```

Comment:
same here...

Reply:
Because with the current state the logic is to fallthrough and I was adding it where it was falling anyway. I was assuming breaks were forgotten, too, and then I've been told not to assume since the fallthrough were intentional. Some of them weren't but most were. Where they are forgotten I've added them like in here:
https://github.com/cms-sw/cmssw/pull/27856
but in other places I've been told otherwise:
https://github.com/cms-sw/cmssw/pull/27811#issuecomment-523469488
Solution was to keep the existing logic until told to change it to break by responsible's.

Reply:
@mrodozov thanks, I see the reasoning behind, but I would like nevertheless to ask @rekovic to check and confirm that this is the intended behaviour of the code